frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/top-consumers-card/utils.ts

```diff
@@ -18,10 +22,10 @@ export const getGraphVectorStats: GetStats = (response, metricType, unit) => {
 };
 
 export const sortResources: SortResourcesProps = (a, b) => {
-  const aVal = _.get(a, 'values');
-  const bVal = _.get(b, 'values');
-  const x = _.get(a, ['values', aVal.length - 1, 1]);
-  const y = _.get(b, ['values', bVal.length - 1, 1]);
+  const aVal = a?.values;
+  const bVal = b?.values;
+  const x: number = parseInt(a?.values?.[aVal.length - 1]?.[1], 10);
+  const y: number = parseInt(b?.values?.[bVal.length - 1]?.[1], 10);
```

Comment:
The default value is 1.
```suggestion
  const x: number = a?.values?.[aVal.length - 1] ?? 1;
```

Reply:
@afreen23  We have to keep parseInt(). Or else it shows type errors. And the second parameter 10 is also important to avoid radix eslint error.